packages/ckeditor5-engine/src/model/differ.ts

```diff
@@ -1094,22 +1096,32 @@ export default class Differ {
 	 *
 	 * @param parent The element in which the change happened.
 	 * @param offset The offset at which change happened.
-	 * @param elementSnapshot The snapshot of the removed element a character.
+	 * @param currentElementSnapshot Map of attributes of the inserted element
+	 * @param removedElementSnapshot Map of the attributes of the removed elements before all changes
+	 * @param action Indicates if insert is part of operation that consist multiple suboperations.
```

Comment:
I updated the comment. I'd stay with keeping only `refresh` and `rename` in `action` optional property. It'll reduce the total amount of modified code, and having these values in `action` is not especially useful at all because we already know the type of operation.